pages/reference/base-images/legacy-base-images.md

```diff
@@ -89,7 +89,7 @@ There is an entry script (`/usr/bin/entry.sh`) as the ENTRYPOINT in every base i
 * Currently, we support 6 architectures: amd64, i386, aarch64, armv7hf, armv6hf and armel; however, not all supported Linux distros support these architectures, only Debian does. We don't have Alpine Linux base images for aarch64 and armel and only armv7hf Fedora base images supported at the moment.
 * The `latest` tag points to Jessie (Debian), 25 (Fedora) and 3.6 (Alpine Linux).
 * If you have your own systemd service and you want your systemd service to use the environment variables you set in the dashboard, you need to add `EnvironmentFile=/etc/docker.env` to your systemd service unit file.
-* ~~To keep the base images fresh, we re-build all base images every week and update version of language specific base images with the latest version in official repository. There are times when the newly built base images could accidentally break your build. To prevent this, we've introduced the date fixed tag, e.g. `jessie-20160510`. The date fixed tag images will never change, so they can be used for a stable build.~~ These images are no longer actively maintained and will receive no future updates.
+* ~~To keep the base images fresh, we re-build all base images every week and update version of language specific base images with the latest version in official repository. There are times when the newly built base images could accidentally break your build. To prevent this, we've introduced the date fixed tag, e.g. `jessie-20160510`. The date fixed tag images will never change, so they can be used for a stable build.~~ These images are no longer actively maintained and will receive no future updates. For the latest images, you should switch to the new [balenalib](/reference/base-images/base-images) base images.
```

Comment:
I realise this sentence didn't change in this commit but fixing it anyway.

```suggestion
* ~~To keep the base images fresh, we re-build all base images every week and update the version of the language specific base images with the latest version available in the official repository. There are times when the newly built base images could accidentally break your build. To prevent this, we've introduced the date fixed tag, e.g. `jessie-20160510`. The date fixed tag images will never change, so they can be used for a stable build.~~ These images are no longer actively maintained and will receive no future updates. For the latest images, you should switch to the new [balenalib](/reference/base-images/base-images) base images.
```

Reply:
@chrisys fixed it up in https://github.com/balena-io/docs/pull/1017/commits/a2b7739c26e1ff4f7dc6f49cabe7649dc9cda8f3